rules/okta_rules/okta_user_account_locked.py

```diff
@@ -0,0 +1,16 @@
+from panther_base_helpers import okta_alert_context
+
+
+def rule(event):
+    return event.get("eventtype") in ("user.account.lock", "user.account.lock.limit")
+
+
+def title(event):
+    return (
+        f"Okta: [{event.get('actor', {}).get('alternateId', '<id-not-found>')}] "
+        "has had their account locked."
```

Comment:
if the account is locked by user-based actions, I might rephrase the verbiage here to be `Okta: [user] [{event.get('displaymessage')}]`. 

The current text leaves me thinking an administrator locked the user's account.